### PR TITLE
Add Tag entities

### PR DESCRIPTION
**Pull request by gjohansson-ST**
## Proposed change

Add `event` entities for `tag` integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/115048
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/tag.markdown [hunk 1]
```diff
@@ -106,6 +108,13 @@ Happy printing!
 
 ![NFC Cards](/images/blog/2020-09-15-home-assistant-tags/cards.jpg)
 
+## Entities
+
+Every card is automatically creating an `event` entity which is useful for automations or to put on dashboards to see when the card was latest scanned.
```
Comment (c0ffeeca7) [inline source/_integrations/tag.markdown]:
```suggestion
Every card automatically creates an `event` entity. This is useful for automations or for showing on dashboards to see when the card was last scanned.
```

File: source/_integrations/tag.markdown [hunk 2]
```diff
@@ -106,6 +108,13 @@ Happy printing!
...
+
+State is the last datetime when the card was scanned.
+`tag_id` and `device_id` is set as attributes on the entity.
```
Comment (c0ffeeca7) [inline source/_integrations/tag.markdown]:
```suggestion
`tag_id` and `device_id` are set as attributes on the entity.
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @gjohansson-ST :+1:

File: source/_integrations/tag.markdown [hunk 3]
```diff
@@ -106,6 +108,13 @@ Happy printing!
 
 ![NFC Cards](/images/blog/2020-09-15-home-assistant-tags/cards.jpg)
 
+## Entities
+
+Every card automatically creates an `event` entity. This is useful for automations or for showing on dashboards to see when the card was last scanned.
+
+State is the last datetime when the card was scanned.
```
Comment (c0ffeeca7) [inline source/_integrations/tag.markdown]:
```suggestion
State is the timestamp when the card was last scanned.
```

**Comment by c0ffeeca7** [comment]
Are you still working on this, @gjohansson-ST? Asking because the PR is marked as draft.

File: source/_integrations/tag.markdown (hunk 3 above)
Comment (frenck) [reply source/_integrations/tag.markdown]: I think datetime is intentional here, as it is a datetime object and not a timestamp string.

**Comment by gjohansson-ST** [comment]
> Are you still working on this, @gjohansson-ST? Asking because the PR is marked as draft.

@c0ffeeca7 sorry but I seem to have forgotten to finalize this PR. I will do it now so it can be finalized shortly 👍

File: source/_integrations/tag.markdown [hunk 4]
```diff
@@ -106,6 +108,13 @@ Happy printing!
 
 ![NFC Cards](/images/blog/2020-09-15-home-assistant-tags/cards.jpg)
 
+## Entities
+
+Every card automatically creates an `tag` entity. This is useful for automations or for showing on dashboards to see when the card was last scanned.
+
+State is the datetime when the card was last scanned.
```
Comment (c0ffeeca7) [inline source/_integrations/tag.markdown]:
```suggestion
State shows the time when the card was last scanned in datetime format. For example, `2013-09-17 07:32:51`.
```

File: source/_integrations/tag.markdown [hunk 5]
```diff
@@ -39,6 +41,17 @@ Home Assistant has a dedicated panel that allows you to manage your tags. You ca
 
 ![Tag user interface in Home Assistant](/images/blog/2020-09-15-home-assistant-tags/tag-ui.gif)
 
+## Entities
+
+Every card automatically creates an `tag` entity. This is useful for automations or for displaying on dashboards to see when the card was last scanned.
+
+State shows the time when the card was last scanned in datetime format. For example, `2013-09-17 07:32:51`
```
Comment (c0ffeeca7) [inline source/_integrations/tag.markdown]:
```suggestion
State shows the time when the card was last scanned in datetime format. For example, `2013-09-17 07:32:51`.
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @gjohansson-ST :+1:

File: source/_integrations/tag.markdown (hunk 3 above)
Comment (frenck) [reply source/_integrations/tag.markdown]: Hmm the state is not a datetime object actually but a timestamp string... so your suggestion is actually correct.

File: source/_integrations/tag.markdown [hunk 6]
```diff
@@ -39,6 +41,17 @@ Home Assistant has a dedicated panel that allows you to manage your tags. You ca
 
 ![Tag user interface in Home Assistant](/images/blog/2020-09-15-home-assistant-tags/tag-ui.gif)
 
+## Entities
+
+Every card automatically creates an `tag` entity. This is useful for automations or for displaying on dashboards to see when the card was last scanned.
+
+State shows the time when the card was last scanned in datetime format. For example, `2013-09-17 07:32:51`.
```
Comment (c0ffeeca7) [inline source/_integrations/tag.markdown]:
```suggestion
State shows the time when the card was last scanned in datetime string format. For example, `2013-09-17T07:32:51.095+00:00`
```
I see. I think this should fix the example?

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**